app/javascript/pages/admin/Tags/index.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react'
+import { graphql, compose } from 'react-apollo'
+import { connect } from 'react-redux'
+import { NetworkStatus } from 'apollo-client'
+import R from 'ramda'
+import ReactTable from 'react-table'
+import { Link } from 'react-router'
+import Dialog from 'material-ui/Dialog'
+import { defaultFilterMethod } from 'lib/utils'
+
+import { graphQLError, togglePopover } from 'actions'
+
+import Loading from 'components/LoadingIcon'
+
+import TagsQuery from './queries/index.gql'
+import DeleteTagMutation from './mutations/delete.gql'
+
+import s from './main.css'
+
+import 'style-loader!css-loader!react-table/react-table.css'
```

Comment:
I usually like to consistently organise my imports, it makes reading easier imo. We might need to come up with some standards and conventions for the portal going forward. But my convention here is to group and order them like so:

all react imports
newline
all third party imports
newline
all local imports